src/App.js

```diff
@@ -1,8 +1,31 @@
 import React from 'react';
 import VoteView from './VoteView';
+import { Route } from 'react-router-dom';
+import SignUp from './SignUp';
+import SignIn from './SignIn';
+import { useCookies } from 'react-cookie';
 
 function App() {
-  return <VoteView />;
+  const [loginCookie, setLoginCookie, removeLoginCookie] = useCookies([
+    'loginCookie',
+  ]);
+
+  return (
+    <div>
+      <Route exact path="/">
+        <VoteView
+          loginCookie={loginCookie}
+          removeLoginCookie={removeLoginCookie}
+        ></VoteView>
+      </Route>
+      <Route path="/signup">
+        <SignUp></SignUp>
+      </Route>
+      <Route path="/signin">
+        <SignIn setLoginCookie={setLoginCookie}></SignIn>
+      </Route>
```

Comment:
```suggestion
      <Route path="/signup">
        <SignUpPage></SignUpPage>
      </Route>
      <Route path="/signin">
        <SignInPage setLoginCookie={setLoginCookie}></SignInPage>
      </Route>
```

보통 Page 컴포넌트들은 네이밍을 Page라고 붙여놓는게 일반적이라고 하더라!